bin/behat-prepare.sh

```diff
@@ -71,10 +71,10 @@ rm -rf $PREPARE_DIR/wp-content/plugins/wp-saml-auth/.git
 # SimpleSAMLphp is installed to ~/code/private, and then symlinked into the
 # web root
 ###
-echo "Setting up SimpleSAMLphp"
+echo "Setting up SimpleSAMLphp 2.4.0"
 rm -rf $PREPARE_DIR/private
 mkdir $PREPARE_DIR/private
-wget https://github.com/simplesamlphp/simplesamlphp/releases/download/v1.18.4/simplesamlphp-1.18.4.tar.gz -O $PREPARE_DIR/simplesamlphp-latest.tar.gz
+wget https://github.com/simplesamlphp/simplesamlphp/releases/download/v2.4.0/simplesamlphp-2.4.0-full.tar.gz -O $PREPARE_DIR/simplesamlphp-latest.tar.gz
```

Comment:
optional: there's some work in #395 handling this with package updater

Reply:
I think I'm going to come back to #395 later and try to integrate that stuff. I don't know that that even needs to go into the release (other than the tested up to bump) because I think it's mostly CI stuff, so my priority with this PR is to get the Behat tests running and passing on this branch.